docs/howto/graph-api/misc.md

```diff
@@ -0,0 +1,319 @@
+         "name":"Theodore Chang",
+         "affiliation":"Humboldt-Universität zu Berlin",
+         "user_id":"57aaf068-cdd0-43c1-be51-99e0d425c131",
+         "created":"2022-02-12T22:14:25.151000+00:00",
+         "is_admin":false,
+         "is_oasis_admin":false
+      }
+   }
+}
+```
+
+In most cases, one shall only get the user information of the user who is currently logged in.
+This can be done conveniently by using the special token `me`.
+Thus, the above query is equivalent to the following query if `57aaf068-cdd0-43c1-be51-99e0d425c131` is the user ID of the currently logged in user.
+
+```json hl_lines="3"
+{
+   "users":{
+      "me":{
+         "m_request":{
+            "directive":"plain"
+         }
+      }
+   }
+}
+```
+
+Starting from user information, it is possible to navigate to other resources (nodes).
+For example, the following query will list all uploads of the currently logged in user.
+
+```json hl_lines="4"
+{
+   "users":{
+      "me":{
+         "uploads":{ "m_request":{ "directive":"plain" } }
+      }
+   }
+}
+```
+
+The response may look like the following.
+
+```json
+{
+  "users": {
+    "me": {
+      "uploads": {
+        "mBMHmsQuSgmgBuBlqaV5fQ": "mBMHmsQuSgmgBuBlqaV5fQ",
+        "G7SMnJwZS5ajTT3eBROM-A": "G7SMnJwZS5ajTT3eBROM-A",
+        "nMwp7V9cQJCZoCkA6Op8xg": "nMwp7V9cQJCZoCkA6Op8xg"
+      }
+    }
+  }
+}
+```
+
+Note it's nothing more than a plain list.
+If one wishes, one can navigate to the corresponding upload to get more information with specific upload IDs, or with a wildcard `*`.
+
+```json hl_lines="5-7"
+{
+   "users":{
+      "me":{
+         "uploads":{
+            "*":{
+               "upload_create_time":{ "m_request":{ "directive":"plain" } }
+            }
+         }
+      }
+   }
+}
+```
+
+The above query will return the creation time of all uploads of the currently logged in user.
+
+```json
+{
+   "users":{
+      "me":{
+         "uploads":{
+            "mBMHmsQuSgmgBuBlqaV5fQ":{
+               "upload_create_time":"2025-06-18T01:23:15.276000"
+            },
+            "G7SMnJwZS5ajTT3eBROM-A":{
+               "upload_create_time":"2025-06-23T15:37:14.401000"
+            },
+            "nMwp7V9cQJCZoCkA6Op8xg":{
+               "upload_create_time":"2025-06-23T16:07:53.100000"
+            }
+         }
+      }
+   }
+}
+```
+
+## Elasticsearch Metadata
+
+Some of the entry metadata is also stored in `Elasticsearch` indices.
+They are similar to the metadata in the MongoDB database, but with more information.
+To fetch data from the `Elasticsearch` index, one can use the special token `search`.
+The top-level request configuration also supports the `query` field.
+This `query` field shall take a valid `Metadata` query object (which itself also contains a `query` field), see the endpoint `/entries/query` for more details.
+
+The following example lists all entries created in 2025 that are visible to the logged in user.
```

Comment:
The used query means more specifically: "... all entries created after the beginning of year 2025"